### PR TITLE
Added NotTheSmoooze; Roguish Archetypes

### PR DESCRIPTION
**Pull request by CorrTerek**
I've added NotTheSmoooze's Roguish Archetypes. This is my first time submitting a pull request. I hope I have done so correctly.

### CODE REVIEW

**Review by TheGiddyLimit — APPROVED** [review]
Looks about right to me 👍

File: subclass/NotTheSmoooze; RoguishArchetypes.json [hunk 1]
```diff
@@ -0,0 +1,763 @@
...
+			"className": "Rogue",
+			"classSource": "PHB",
+			"subclassShortName": "Clover",
+			"subclassSource": "NotTSRoguishArchetypes",
+			"level": 3,
+			"entries": [
+				"Clovers are rogues with supernatural fortune. You might believe your luck to be the blessing of the gods, the favour of your ancestors, or simply your birthright. Alternatively, many clovers are highly superstitious; you might think your luck is owed to simple, ritualistic habits. At your option, you can roll on the Superstitions table below to create a quirk.",
+				{
+					"type": "table",
+					"name": "Superstitions",
```
Comment (TheGiddyLimit) [inline subclass/NotTheSmoooze; RoguishArchetypes.json:227]: this shouldn't have a `name`/`source`/`page`

File: subclass/NotTheSmoooze; RoguishArchetypes.json [hunk 2]
```diff
@@ -0,0 +1,763 @@
...
+			"className": "Rogue",
+			"classSource": "PHB",
+			"subclassShortName": "Mindhunter",
+			"subclassSource": "NotTSRoguishArchetypes",
+			"level": 3,
+			"entries": [
+				"Starting at 3rd level, you can use your psychic power to cast a spell. The spell must be a {@filter divination|spells|class=wizard|school=D} or {@filter enchantment|spells|class=wizard|school=E} spell on the wizard spell list. The maximum level of spell that you can cast, as well as the level you cast it at, is determined by your rogue level as shown on the Psychic Casting table below. You can cast this spell without using any components.",
+				"Once you use this feature, you can't use it again until you finish a short or long rest. Starting when you reach 13th level, you can use this feature twice between rests.",
+				{
+					"type": "table",
```
Comment (TheGiddyLimit) [inline subclass/NotTheSmoooze; RoguishArchetypes.json:369]: same here

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]
I take it all back; the automated tests hadn't finished 🥴

File: subclass/NotTheSmoooze; RoguishArchetypes.json (hunk 2 above)
Comment (CorrTerek) [reply subclass/NotTheSmoooze; RoguishArchetypes.json:369]: I'll get right on that.

Comment (CorrTerek) [reply subclass/NotTheSmoooze; RoguishArchetypes.json:366]: Should be good now.

File: subclass/NotTheSmoooze; RoguishArchetypes.json [hunk 3]
```diff
@@ -0,0 +1,757 @@
...
+		},
+		{
+			"name": "Mycologist",
+			"source": "NotTSRoguishArchetypes",
+			"className": "Rogue",
+			"classSource": "PHB",
+			"subclassShortName": "Mycelian",
+			"subclassSource": "NotTSRoguishArchetypes",
+			"weaponProficiencies": [{
+				"blowgun": true
```
Comment (TheGiddyLimit) [inline subclass/NotTheSmoooze; RoguishArchetypes.json:500]: needs to be `blowgun|phb`

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]

**Pull request merged**